<launch>
  <node pkg="test_svo2" type="demo_play_video" name="svo" clear_params="true" output="screen">
    <param name="calib_file" value="$(find test_svo2)/calib/mynteye_mono_calib.yaml" />
    <rosparam file="$(find test_svo2)/param/mynteye_mono.yaml" />
  </node>

  <!--node name="vis" pkg="rviz" type="rviz" args=" -d $(find test_svo2)/rviz_config.rviz" /-->

</launch>
